test/sqllogictest/chbench.slt

```diff
@@ -1613,3 +1613,27 @@ ORDER BY substr(c_state, 1, 1)
 Finish order_by=(#0 asc) limit=none offset=0 project=(#0..#2)
 
 EOF
+
+query T multiline
```

Comment:
How is this change related? :)

Reply:
Oops. In my attempt to rebase this branch on an earlier version of main (since there's that build error), I appear to have accidentally checked in test code from another branch.